gems/pending/blackbox/VmBlackBox.rb

```diff
@@ -88,7 +88,7 @@ def deleteLocalDataDir(options={})
     end
 
     def writeData(filename, data)
-      Dir.mkdir(@localDataDir, 755) unless File.exist?(@localDataDir)

```

Comment:
This code is in use, it just doesn't write into the black box any longer - it stashed the info in the filesystem.